apps/admin_panel/assets/src/omg-page-members/index.js

```diff
@@ -0,0 +1,210 @@
+import React, { Component } from 'react'
+import TopNavigation from '../omg-page-layout/TopNavigation'
+import styled from 'styled-components'
+import SortableTable from '../omg-table'
+import { Button, Icon } from '../omg-uikit'
+import MembersFetcher from '../omg-member/membersFetcher'
```

Comment:
theres a compile warning from here.. should capitalize the `/MembersFetcher` as the file is